storage_service/administration/views.py

```diff
@@ -27,7 +28,14 @@
 # ######################## ADMIN ##########################
 
 
-def settings_edit(request):
+def index(request):
+    if request.user.is_superuser:
```

Comment:
While testing this I noticed `Managers` can access the `Configuration` view but this redirects them to `Users` instead.

Reply:
Good catch, I'll use this instead:

    request.user.has_perm("administration.change_settings")